python/ray/autoscaler/__init__.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import logging
+
+
+def _setup_logger():
+    logger = logging.getLogger("ray.autoscaler")
```

Comment:
Should we just have a setup logger for the ray module instead of for each submodule?

Reply:
Basically, if ray.init() is called with setup_logger then do setup, otherwise don't. And CLI would default to having setup=True, and we remove all the basicConfig() stuff.

Reply:
@robertnishihara @pcmoritz

Reply:
Having one logger for the ray module makes sense I think (certainly for the autoscaler, I could see a case for having different ones for tune/rllib/other libraries).  We also need to make sure that it doesn't clash with the user's logging.

Reply:
ok I implemented @ericl's suggestion. can someone try this out?

Reply:
Done .